Sources/WalletConnectSign/Serializing.swift

```diff
@@ -2,9 +2,17 @@
 import Foundation
 import WalletConnectKMS
 
-public protocol Serializing {
-    func serialize(topic: String, encodable: Encodable) throws -> String
-    func tryDeserialize<T: Codable>(topic: String, message: String) -> T?
+protocol Serializing {
+    func serialize(topic: String, encodable: Encodable, envelopeType: Envelope.EnvelopeType) throws -> String
+    func tryDeserialize<T: Codable>(topic: String, encodedEnvelope: String) -> T?
+}
+
+extension Serializing {
+    func serialize(topic: String, encodable: Encodable) throws -> String {
+        try serialize(topic: topic, encodable: encodable, envelopeType: .type0)
+    }
+
 }
 
 extension Serializer: Serializing {}
```

Comment:
Maybe move protocol conformance to Serializer?

Reply:
yeah, that was error prone